packages/nextjs/components/scaffold-eth/Address.tsx

```diff
@@ -39,6 +41,20 @@ export const Address = ({ address, disableAddressLink, format }: TAddressProps)
   useEffect(() => {
     setEnsAvatar(fetchedEnsAvatar);
   }, [fetchedEnsAvatar]);
+  useEffect(() => {
+    const sizeMap = {
+      "3xl": 10,
+      "2xl": 9,
+      xl: 8,
+      lg: 7,
+      base: 5,
+      sm: 4,
+      xs: 3,
+    };
+    setEnsAvatarSize((sizeMap[size] * 24) / sizeMap["base"]);
+
+    setBlockiesSize(sizeMap[size]);
+  }, [size]);
```

Comment:
Why do we need useState / useEffect here?

I think we can just calculate the size directly.

Reply:
Done, removed useEffect and useState